### PR TITLE
Update SPM Client to 2.0.0 version, from Prestashop extension - refs #60

### PR DESCRIPTION
**Pull request by real34**

### CODE REVIEW

File: src/lib/ShopymindClient/Src/Reminders/index.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/Reminders/index.php]: CS : newline at file end

File: src/lib/ShopymindClient/Src/index.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Controller/Request/index.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Controller/Request/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Controller/index.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Controller/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Http/Client/Adapter/index.php [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Http/Client/Adapter/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Http/Client/index.php [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Http/Client/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Http/index.php [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Http/index.php]: CS: newline at file end

File: src/lib/ShopymindClient/Src/library/SZend/Json/index.php [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+			    	
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+						
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+						
+header("Location: ../");
+exit;
```
Comment (adrien-louis-r) [inline src/lib/ShopymindClient/Src/library/SZend/Json/index.php]: CS: newline at file end

**Comment by adrien-louis-r** [comment]
:+1:

**Comment by damien-biasotto** [comment]
:+1

**Pull request merged**